### DISCUSSION TITLE
Removing datafeeder from gn/gn-ui

### DISCUSSION THREAD

**Discussion by f-necas** [geonetwork/geonetwork-ui#917] [General]
Datafeeder seems (to me) to be more a geOrchestra tool than a geonetwork tool.

As the [backend](https://github.com/georchestra/georchestra/tree/master/datafeeder) is in geOrchestra's repo, I'm questionning myself about the interest in keeping it here.

WDYT of it ?

**Comment by jahow** [comment]
I think this can be seriously considered for the next release and I am in favor of it. Please keep in mind that this will probably incur additional maintenance by the geOrchestra community (e.g. security fixes, dependencies and framework updates...).

**Comment by jahow** [comment]
Closing as this has been done
